docs-ref-conceptual/includes/cli-install-linux-dnf.md

```diff
@@ -35,21 +39,22 @@ for the Azure CLI. This package has been tested with RHEL 7.7, RHEL 8, Fedora 24
    gpgkey=https://packages.microsoft.com/keys/microsoft.asc" | sudo tee /etc/yum.repos.d/azure-cli.repo
```

Comment:
@Svijay-msft Should this still be `yum.repos.d`?

Reply:
DNF -  by default uses all *.repo files found under /etc/yum.repos.d. So we should be good having **yum.repos.d**  .